src/sass/_Fabric.Icons.scss

```diff
@@ -347,3 +347,20 @@
     z-index: $ms-zIndex-back;
   }
 }
+
+// Fabric Core Icon variables
+$ms-Icon-12: 12px;
+$ms-Icon-16: 16px;
+$ms-Icon-20: 20px;
```

Comment:
Also, we should be using $ms-icon-s, $ms-icon-m, $ms-icon-l since we are prescribing a ramp.